content/docs/sidebar.json

```diff
@@ -488,6 +488,10 @@
     "label": "Python API Reference",
     "source": "api-reference/index.md",
     "children": [
+      {
+        "slug": "params_show",
+        "label": "params_show()"
+      },
       {
         "slug": "get_url",
```

Comment:
Is there any sorting to this list BTW? Unclear to me why this comes first. If it's by relevance, this should probably be 2nd to last?

Reply:
My bad, I didn't think of the order. 

> Is there any sorting to this list BTW?

I think the usual sort is alphabetical

Reply:
Yeah up to you.

Loosely related: should https://dvc.org/doc/api-reference be alphabetical too? (just move `make_checkpoint` up a little)